modules/ROOT/pages/troubleshoot-saml-assertions-task.adoc

```diff
@@ -34,10 +34,10 @@ https://www.mulesoft.com/support-login
 
 == Example: Chrome Developer Tools SAML Response Element
 
-The following image shows a SAML response element in Chrome developer tools as a result of logging in to Anypoint Platform using SSO.
+The following image shows a SAML response element in Chrome developer tools as a result of signing in to Anypoint Platform using SSO.
 
-.SAML response element after logging in to Anypoint Platform in Google Chrome.
+.SAML response element after signing in to Anypoint Platform in Google Chrome.
```

Comment:
We don't use captions anymore. Instead, we use alt text...

Reply:
Yeah. I'll clean this up later. This was only a S/R to change the log ins to sign ins. There is work that needs to be done for the images and alt text. I'll do that as part of the screen shot audit.